### PR TITLE
Custom fees

### PR DESCRIPTION
**Pull request by codingwithredbull**
Added custom fees option to wallet

### CODE REVIEW

File: MainWindow.glade [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.18.3 -->
 <interface>
-  <requires lib="gtk+" version="3.12"/>
+  <!-- interface-requires gtk+ 3.12 -->
```
Comment (CaptainMeatloaf) [inline MainWindow.glade]: Why was this changed?

File: MainWindow.py [hunk 2]
```diff
@@ -147,6 +156,22 @@ def on_SendButton_clicked(self, object, data=None):
             self.builder.get_object("TransactionStatusLabel")\
                 .set_label("Slow down TRTL bro! The amount needs to be a number greater than 0.")
             return
+            
+        #Determine Fee Settings
+        feeSuggest = self.builder.get_object("FeeSuggestionCheck")
+        if not feeSuggest.get_active():
```
Comment (CaptainMeatloaf) [inline MainWindow.py]: This needs more comments, explain what you are doing here

File: MainWindow.glade (hunk 1 above)
Comment (RocksteadyTC) [reply MainWindow.glade]: I might be wrong but I think this is the accepted method to express this require. I'm referencing code from the gtk docs here. http://python-gtk-3-tutorial.readthedocs.io/en/latest/builder.html

File: MainWindow.py (hunk 2 above)
Comment (RocksteadyTC) [reply MainWindow.py]: My observation: This is the toggle for "suggested fee" on the wallet. If the suggested fee isnt checked, use the one that is defined in the fee box.

File: MainWindow.glade (hunk 1 above)
Comment (CaptainMeatloaf) [reply MainWindow.glade]: This was initially auto-generated by glade's GUI designer - it seemed to work fine before, I'm just wondering why it was changed

File: MainWindow.py (hunk 2 above)
Comment (CaptainMeatloaf) [reply MainWindow.py]: Indeed, however this is not explained with comments, and a typical user unfamiliar with glade may struggle to understand what is going on here.
